### PR TITLE
Added title to review creation code and UI / Refactored to camelCase

### PR DESCRIPTION
**Pull request by mbouilloud**
@AlexandreMessmer I changed your UI to add a title for reviews, check that you still like it :)

### CODE REVIEW

File: app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt [hunk 1]
```diff
@@ -60,6 +62,7 @@ class AddReviewActivity : AppCompatActivity() {
 
         ratingBar = findViewById(R.id.reviewRatingBar)
```
Comment (jmettler27) [inline app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt:63]: The two next lines have ids with snake_case. We will need to decide whether snake_case or camelCase is appropriate for ids, but apart from that everything seems ok.

**Review by jmettler27 — APPROVED** [review]

File: app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt (hunk 1 above)
Comment (mbouilloud) [reply app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt:63]: Well someone can clean all naming issues and do a PR.

Comment (mbouilloud) [reply app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt:63]: I will try to do it within this PR.

Comment (mbouilloud) [reply app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt:62]: Done, you can check the commit.

Comment (jmettler27) [reply app/src/main/java/com/github/sdp/ratemyepfl/activity/AddReviewActivity.kt:62]: Perfect!

**Review by jmettler27 — APPROVED** [review]

**Review by NicolasMatekalo — APPROVED** [review]
Nothing much to say as these are minor changes. Neverthless, setting title through the UI is indeed a nice feature. Thanks for the refactoring of IDs, we will stick to that from now on

**Pull request merged**